sorc/build_ww3prepost.sh

```diff
@@ -35,6 +34,7 @@ export Z_LIB=${Z_LIB:-$ZLIB_ROOT/lib/libz.a}
 export JASPER_LIB=${JASPER_LIB:-$JASPER_ROOT/lib64/libjasper.a}
 export WWATCH3_NETCDF=NC4
 export NETCDF_CONFIG=$NETCDF_ROOT/bin/nc-config
+export PRINTER=${PRINTER:-""}
```

Comment:
it's a variable that's defined as part of the ww3 build which is getting cleaned out.  So this should go away soon.  I'm assuming Walter added it because a version he was using needed it for some reason?

Reply:
I turned on set -u and the script errored because it wasn't defined. So I defined it.